distro/webjar/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.camunda.bpm</groupId>
+    <artifactId>camunda-parent</artifactId>
+    <relativePath>../../parent</relativePath>
+    <version>7.5.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.camunda.bpm.webapp</groupId>
+  <artifactId>camunda-webapp-webjar</artifactId>
+  <packaging>jar</packaging>
+  <name>camunda BPM - webapp - webjar</name>
+
+  <properties>
+    <skipTests>true</skipTests>
+  </properties>
+
+  <!-- add dependency so this is run after the war was build -->
+  <dependencies>
+    <dependency>
+      <groupId>org.camunda.bpm.webapp</groupId>
+      <artifactId>camunda-webapp</artifactId>
+      <version>${project.parent.version}</version>
+      <type>jar</type>
```

Comment:
Well, if you tell the WebApp to work without the backend... ;)

On 13.04.16 06:51, Falko Menge wrote:
>
> In distro/webjar/pom.xml
> <https://github.com/camunda/camunda-bpm-platform/pull/223#discussion_r59492558>:
>
> > +  <groupId>org.camunda.bpm.webapp</groupId>
> > +  <artifactId>camunda-webapp-webjar</artifactId>
> > +  <packaging>jar</packaging>
> > +  <name>camunda BPM - webapp - webjar</name>
> > +
> > +  <properties>
> > +    <skipTests>true</skipTests>
> > +  </properties>
> > +
> > +  <!-- add dependency so this is run after the war was build -->
> > +  <dependencies>
> > +    <dependency>
> > +      <groupId>org.camunda.bpm.webapp</groupId>
> > +      <artifactId>camunda-webapp</artifactId>
> > +      <version>${project.parent.version}</version>
> > +      <type>jar</type>
>
> Okay, I discovered that the Spring Boot Starter doesn't work, if it
> doesn't have the classes as a transitive dependency from the webjar. I
> guess it might be a little unconventional for a webjar to also
> "contain" Java classes instead of just client-side dependencies, but
> that's fine for now.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/camunda/camunda-bpm-platform/pull/223/files/3d59edca44738c051ebe88a7e4138df0bc78b986#r59492558>
>